docs-ref-conceptual/update-azure-cli.md

```diff
@@ -51,4 +51,15 @@ az config set auto-upgrade.all=no
 If you decide not to use the automatic update feature any more for cases like keeping command scripts running stably, you can turn it off through configuration.
 ```azurecli
 az config set auto-upgrade.enable=no
-```
\ No newline at end of file
+```
+## Update Assistance
+
+The Azure cli also provides a reference command, [az command-change](https://github.com/Azure/azure-cli-extensions/tree/main/src/command-change) and a website, [azure-cli-diff-site](https://azure-cli-diff-site.azurewebsites.net/), to help you examine changes between CLI versions. Use the version comparison list provided to prepare your scripts for a version update. See [Breaking change detection for the Azure CLI](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_detection_guidelines.md) for more information.
```

Comment:
The links provided here should point to the conceptual and reference documentation associated with the command.